src/pages/LogInWithShortLivedAuthTokenPage.js

```diff
@@ -105,9 +122,4 @@ LogInWithShortLivedAuthTokenPage.propTypes = propTypes;
 LogInWithShortLivedAuthTokenPage.defaultProps = defaultProps;
 LogInWithShortLivedAuthTokenPage.displayName = 'LogInWithShortLivedAuthTokenPage';
 
-export default compose(
-    withLocalize,
-    withOnyx({
-        account: {key: ONYXKEYS.ACCOUNT},
-    }),
-)(LogInWithShortLivedAuthTokenPage);
+export default compose(withLocalize, withOnyx({account: {key: ONYXKEYS.ACCOUNT}}), withOnyx({session: {key: ONYXKEYS.SESSION}}))(LogInWithShortLivedAuthTokenPage);
```

Comment:
As this is function component, we can use `useLocalize` hook.

Is there a problem using one `withOnyx` HOC? I remember this was fixed recently and no longer need multiple `withOnyx`

Reply:
ah no, just outdated format i think. Updated!